src/dlgautodj.cpp

```diff
@@ -197,3 +202,60 @@ void DlgAutoDJ::setTrackTableFont(const QFont& font) {
 void DlgAutoDJ::setTrackTableRowHeight(int rowHeight) {
     m_pTrackTableView->setTrackTableRowHeight(rowHeight);
 }
+
+void DlgAutoDJ::updateSelectionInfo()
+{
+    int duration = 0;
+    
+    QModelIndexList indices = m_pTrackTableView->selectionModel()->selectedRows();
+
+    for(int i=0;i!= indices.size();i++) {
+      TrackPointer pTrack = m_pAutoDJTableModel->getTrack(indices.at(i));
+      if (pTrack) {
+        duration += pTrack->getDuration();
+      }
+    }
+
+    QString label = QString();
+    QString tooltip = QString();
+    
+    if (!indices.isEmpty()) {
+    	
+      	int hours = duration / 3600;
+	int minutes = (duration - (hours*3600)) / 60;
+	int seconds = duration - (hours * 3600) - (minutes*60);
+
```

Comment:
You don't need to, I'm quite agnostic on these subjects (but I'm not sure what the others will say). Try, at least, to add spaces around all operators please.